pulp_ansible/app/viewsets.py

```diff
@@ -183,6 +310,49 @@ class CollectionVersionViewSet(UploadGalaxyCollectionMixin, SingleArtifactConten
     serializer_class = CollectionVersionSerializer
     filterset_class = CollectionVersionFilter
     ordering_fields = ("pulp_created", "name", "version", "namespace")
+    queryset_filtering_required_permission = "ansible.view_collection"
+
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list", "retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+            },
+            {
+                "action": "create",
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_model_perms:ansible.add_collection",
+            },
+            {
+                "action": "destroy",
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": [
+                    "has_model_or_obj_perms:ansible.delete_collection",
+                    "has_model_or_obj_perms:ansible.view_collection",
+                    "has_model_or_obj_perms:ansible.change_ansiblerepository",
```

Comment:
same question as the collections endpoint

Reply:
See my answer from collections endpoint. 

@bmclaughlin Collections Version doesn't have a destroy or update endpoint.